Lexplorer/Pages/AccountDetails.razor

```diff
@@ -175,13 +209,25 @@
     public bool isLoading = true;
     public readonly int pageSize = 25;
     public readonly int nftPageSize = 12; //6 per row
+    public int totalNfts = 0;
+    public int nftPages = 0;
 
     private Account? account { get; set; }
     private IList<Transaction>? transactions { get; set; } = new List<Transaction>();
     private IList<AccountNFTSlot>? accountNFTSlots { get; set; }
     private Dictionary<string, NftMetadata> NFTdata { get; set; } = new Dictionary<string, NftMetadata>();
     private CancellationTokenSource? cts;
 
+    protected override async Task OnInitializedAsync()
+    {
+        // Method for getting the total number of NFTS for account
+        string totalNftsKey = $"account{accountId}-totalnfts";
+        totalNfts = await AppCache.GetOrAddAsyncNonNull(totalNftsKey, 
+            async () => await LoopringGraphQLService.GetAccountTotalNfts(accountId!), 
+            DateTimeOffset.UtcNow.AddMinutes(10));
+        nftPages = (int)Math.Ceiling(decimal.Divide(totalNfts, nftPageSize));
+    }
+
```

Comment:
If you change accountId either in url manually or by clicking to/from to get to a new account it would reload the page and OnInitializedAsync would be triggered again. So that would not be an issue, but I will relocate it to OnParameter... :)

Reply:
It has been moved to OnParameter.... So this is ok now :)